src/cts/test/max_cap.tcl

```diff
@@ -4,7 +4,15 @@ read_lef sky130hs/sky130hs.tlef
 read_lef sky130hs/sky130hs_std_cell.lef
 read_liberty sky130hs/sky130hs_tt.lib
 
-set def_file "max_cap.def"
+if {[info exists ::env(TEST_TMPDIR)] && [llength $::env(TEST_TMPDIR)] > 0} {
+  # Bazel tmp dir per test
+  set tmpdir [lindex $::env(TEST_TMPDIR) 0]
```

Comment:
what does `lindex` in this case ? Isn't the environment variable already storing exactly one element ?